src/robot/variables/typeconverters.py

```diff
@@ -122,13 +124,33 @@ def _literal_eval(self, value, expected):
         return value
 
 
+@TypeConverter.register
+class StringConverter(TypeConverter):
+    type = str
+    type_name = 'string'
+    aliases = ('str',)
+
+    def _convert(self, value, explicit_type=True):
+        return str(value)
+
+
 @TypeConverter.register
 class BooleanConverter(TypeConverter):
     type = bool
     type_name = 'boolean'
     aliases = ('bool',)
 
     def _convert(self, value, explicit_type=True):
+        try:
+            return self._string_convert(value)
+        except:
+            try:
+                return bool(value)
+            except:
+                pass
+        raise ValueError
```

Comment:
Why is this code needed?

Should never use plain `except:`, unless the exception handling code checks the exception type, to avoid hiding `SystemExit`, `KeyboardInterrupt`, and other such exceptions.

Reply:
I split the PR after you request, this part of code is not anymore